.github/workflows/go-test.yaml

```diff
@@ -70,6 +70,14 @@ jobs:
           set -ex
           kubectl rook-ceph rbd ls replicapool
 
+      - name: Subvolume command
+        run: |
+          set -ex
+          kubectl rook-ceph ceph fs subvolume create myfs test-subvol group-a 
+          kubectl rook-ceph subvolume ls
+          kubectl rook-ceph subvolume ls --stale
+          kubectl rook-ceph subvolume delete test-subvol myfs group-a
```

Comment:
this should be same as create arg `fsname sub-volname groupname` not the `subvolname fsname and groupname`

Reply:
@yati1998 ^^

Reply:
I will take care

Reply:
@Madhu-1 sorry just forgot, this is in `subvolname fsname and groupname` so that we can add multiple subvolname for same fsname and groupname which will make group deletion easy incase we have many subvols.

Reply:
doesnt seems right to me, we just stick to how we have for create API call, user can automate the CLI call if he /she thinks they need repetitive operations.